docs/stepbystep/react/lesson1.md

```diff
@@ -0,0 +1,81 @@
+# Step by step apps creation with ILC, React, Lesson 1
+
+Hi folks! This document will teach you how to create simple isomorphic React application and run it with ILC in few minutes.
+
+So buckle in your seat belt and let's get started 🚀
+
+## We need React app with SSR first
+
+As you might already guess to build new micro frontend we need to have regular application first. 
+Hopefully I already did all the dirty job for you dear reader. 
+
+So you can go and grab it here https://github.com/StyleT/ilc-learning-react.
```

Comment:
dot symbol includes in the link

Reply:
I see that GitHub renders it correctly https://take.ms/dfLYr

Reply:
try click on the link. don't copy/paste.